php/src/Google/Protobuf/Timestamp.php

```diff
@@ -159,11 +159,11 @@ public function setNanos($var)
     /**
      * Converts PHP DateTime to Timestamp.
      *
-     * @param DateTime $datetime
+     * @param \DateTime $datetime
      */
     public function fromDateTime($datetime)
     {
-        if (get_class($datetime) !== \DateTime::class) {
+        if ($datetime instanceOf \DateTime) {
```

Comment:
I realize it's not case sensitive, but if you want to stick with community convention, it's `instanceof`, not `instanceOf`.

Reply:
Done